### PR TITLE
Add message for cancelled assignment status

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** Add a dedicated status message to display at the top of cancelled assessments. This prevents the "Instructions Unavailable" fallback from showing.

**What changed?**

- Add template and new message for cancelled assignments
- Test refactoring
- Bump ORA to `2.9.10`

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [EDUCATOR-5282](https://openedx.atlassian.net/browse/EDUCATOR-5282)

Screen Shot 2020-09-14 at 13 00 21

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Review by jansenk — APPROVED** [review]
looks great

**Review by jnlapierre — APPROVED** [review]
LGTM and I need to read up on MagicMock.
